frontend/src/lib/utils.ts

```diff
@@ -571,3 +571,11 @@ export function isObviousOwner(path: string, user?: UserExt): boolean {
 	}
 	return false
 }
+
+export function extractCustomProperties(styleStr: string): string {
```

Comment:
@rubenfiszel I was forwarding the entire css style, where I only needed to forward the CSS variables